Sources/FoundationEssentials/FileManager/SearchPaths/FileManager+WindowsSearchPaths.swift

```diff
@@ -0,0 +1,67 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if os(Windows)
+
+private func _url(for id: KNOWNFOLDERID) -> URL {
+    var pszPath: PWSTR?
+    let hResult: HRESULT = withUnsafePointer(to: id) { id in
+        SHGetKnownFolderPath(id, DWORD(KF_FLAG_DEFAULT.rawValue), nil, &pszPath)
```

Comment:
I think it makes sense to just put `KF_FLAG_DEFAULT` as an internal var in `WinSDK+Extensions.swift`